### DISCUSSION TITLE
Can it be used as a mod and not as a bot?

### DISCUSSION THREAD

**Discussion by olzx** [PrismarineJS/mineflayer#1810] [Q&A]
Hello everyone, I really liked this idea, creating bots and interacting with the world using javascript. But the question is: is it possible to run this code on a regular minecraft client, as if it would be a regular mod, but not on minecraft forge, but using the code from this project (mineflayer). In short, I would like to write a javascript mod for minecraft to use it on a server or in a single player game. Is there such a possibility? Maybe someone knows other ways?

**Comment by rom1504** [comment]
You could try to built a mineflayer proxy or maybe use https://github.com/PrismarineJS/prismarine-web-client

**Reply by olzx** [reply]
Thanks, try to figure it out, is there a community of discord for mineflayer?

**Comment by rom1504** [comment]
Yeah, the link is in the readme
